app/javascript/styles/budget_bar.scss

```diff
@@ -9,3 +9,7 @@ $budget-bar-font-size: 14px;
   color: $black !important;
   font-size: $budget-bar-font-size;
 }
+
+#budget_bar {
```

Comment:
I think taking this out is going to be a little more complicated - one of the tough things about maintaining this system is that we have every fund running off the same instance, so changes like this can't be universal or it would shut things off for others too. I would recommend solving this by changing the Config model so that this is toggle-able on and off, with a default of on -- see @lomky 's PR in https://github.com/DARIAEngineering/dcaf_case_management/pull/2343/files for an example of how to do this.

Reply:
AH, ok that makes sense. For some reason I thought different funds were running on different instances. I'll check out that PR, thank you.

Reply:
I made an issue so we can track this element of work specifically - https://github.com/DARIAEngineering/dcaf_case_management/issues/2688